jupyter-docker/run-jupyter-local.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
+#
+# To rebuild the docker image, run:
+#   docker build jupyter-docker/ -t broadinstitute/leonardo-notebooks:local
+set -e
 
-DOCKER_IMG=jupyter/minimal-notebook
+DOCKER_IMG=broadinstitute/leonardo-notebooks:local
```

Comment:
👍 , great to test with our docker image. Spark/Hail won't be available, but cool that it works anyway.